libs/barista-components/chart/src/heatfield/chart-heatfield.html

```diff
@@ -1,6 +1,7 @@
 <button
   class="dt-chart-heatfield-marker"
   [class.dt-chart-heatfield-active]="active"
+  [style.visibility]="_isValidStartEndRange ? 'visible' : 'hidden'"
```

Comment:
Is it possible to click the invisible? Please check that :)
Maybe add an e2e test scenario for that

Reply:
Have already checked that, and here is a stackoverflow thread: https://stackoverflow.com/questions/5659563/css-is-a-hidden-object-clickable

Reply:
As for the e2e test, might add it later ya but not on this pr